test/src/webgl/context-features.spec.js

```diff
@@ -1,6 +1,6 @@
-import {hasFeature, hasFeatures, getFeatures, FEATURES} from 'luma.gl';
+import {canCompileGLGSExtension, hasFeature, hasFeatures, getFeatures, FEATURES} from 'luma.gl';
 import test from 'tape-catch';
-// import sinon from 'sinon';
+import sinon from 'sinon';
```

Comment:
I also want to get rid of the sinon dependency. Opened https://github.com/uber-web/probe.gl/pull/24 as first step.

Reply:
Sounds good- once that's available, I'll update this PR.